### PR TITLE
feat(skin): rename some design tokens

### PR DESCRIPTION
**Pull request by marcoskolodny**
BREAKING CHANGE: some mistica internal tokens are renamed

Renamed tokens: https://github.com/Telefonica/mistica-design/pull/1869/

Issue: [Link](https://jira.tid.es/browse/WEB-1990)

### CODE REVIEW

File: src/__screenshot_tests__/__image_snapshots__/private-skin-components-screenshot-test-tsx-components-in-vivo-new-4-snap.png
Comment (yceballost) [inline src/__screenshot_tests__/__image_snapshots__/private-skin-components-screenshot-test-tsx-components-in-vivo-new-4-snap.png:1]: expected

**Review by yceballost — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Comment by yceballost** [comment]
I suggest to rename the pr title to "rename some internal design tokens" to be more clear about the impact of this change in release notes.

I know that these token can be used in any place, but should not be a common use

**Comment by marcoskolodny** [comment]
> I suggest to rename the pr title to "rename some internal design tokens" to be more clear about the impact of this change in release notes.
> 
> I know that these token can be used in any place, but should not be a common use

I've modified the PR description with your suggestion (that message will be in the release notes)

**Pull request merged**